File: src/test/java/com/ject/studytrip/auth/helper/TokenTestHelper.java
package com.ject.studytrip.auth.helper;

import com.ject.studytrip.auth.infra.provider.TokenProvider;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Component;

@Component
public class TokenTestHelper {

    private final TokenProvider tokenProvider;

    @Autowired
    public TokenTestHelper(TokenProvider tokenProvider) {
        this.tokenProvider = tokenProvider;
    }

    public String createAccessToken(String memberId, String role) {
        return tokenProvider.createAccessToken(memberId, role);
    }

    public String createRefreshToken() {
        return tokenProvider.createRefreshToken();
    }
}

File: src/test/kotlin/com/ject/studytrip/auth/helper/TokenTestHelper.kt
package com.ject.studytrip.auth.helper

import com.ject.studytrip.auth.infra.provider.TokenProvider
import org.springframework.stereotype.Component

@Component
class TokenTestHelper(
    private val tokenProvider: TokenProvider,
) {
    fun createAccessToken(
        memberId: String,
        role: String,
    ): String = tokenProvider.createAccessToken(memberId, role)

    fun createRefreshToken(): String = tokenProvider.createRefreshToken()
}
